config.toml

```diff
@@ -171,6 +171,11 @@ no = 'Sorry to hear that. Please <a href="https://github.com/rancher-sandbox/cOS
 enable = false
 
 [params.links]
+[[params.links.developer]]
+  name = "Slack"
+  url = "https://rancher-users.slack.com/messages/cos-toolkit"
```

Comment:
arent we gonna have a new public space on the new slack? I dont check the older one at all nor the rancher-users or anything else lol